app/assets/javascripts/backbone/views/project_step_view.coffee

```diff
@@ -91,5 +91,15 @@ class MS.Views.ProjectStepView extends Backbone.View
 
   deleteLog: (e, response) ->
     $.post @$(e.target).attr('href'), {_method: 'DELETE'}
+
+    # Number of logs currently visible includes one just deleted from database
+    logsList = @$(e.target).closest('.logs-list')
+    numOfLogs = @$(logsList).children().length
```

Comment:
Please combine the two above lines into one. It's ok to chain a bunch of jquery methods in a row (it's designed for this), as long as readability doesn't suffer.

`@$(e.target).closest('.logs-list').children().length` is quite clear.